.github/workflows/release.yml

```diff
@@ -12,9 +12,6 @@ env:
   CI_COMMIT_AUTHOR: hc-github-team-tf-provider-devex
   CI_COMMIT_EMAIL: github-team-tf-provider-devex@hashicorp.com
 
-permissions:
-  contents: write
-
```

Comment:
Related to the other comment, it would be good to explicitly set this to:

```yaml
permissions:
  contents: read # Write uses service account token
```

Reply:
Fixed ⚡